one-acre-fund/ussd-sms
# Ser 209 order avocado trees

**SER-209: As a client, I want to add avocado trees to my order, so that I can plant a high-impact product**

Added service in RW-core to allow clients to enroll avocado trees


**To test with someone who has 0 as prepayment:**
- Dial and choose 1 for an existing client
- Choose 7 on the next menu for ordering Avoca  
- Use 29174485 as account number **a message should be displayed that they need to pay first**
**To test with someone who has a prepayment of 2000 but needs have 3000 because they ordered 5 Avocado:**
- Dial and choose 1 for an existing client
- Choose 7 on the next menu for ordering Avoca  
- Use 21428746 as account number **a message should be displayed that they need to pay first**
**To test with someone with a high prepayment and place an order:**
- Dial and choose 1 for an existing client
- Choose 7 on the next menu for ordering Avoca  
- Use 21117840 as account number **a message should be displayed that they can order up to 5**
- chose a number
- choose 1 to confirm. An SMS should be sent saying the order is complete

Test link: https://telerivet.com/p/8799a79f/service/SV1161debc042a9de0/edit
